test/functional/tests/data_integrity/test_data_integrity_5d.py

```diff
@@ -0,0 +1,194 @@
+#
+# Copyright(c) 2019 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import random
+import datetime
+
+import pytest
+
+from api.cas import casadm
+from api.cas.cache_config import CacheMode
+from core.test_run import TestRun
+from test_tools import fs_utils
+from test_tools.disk_utils import Filesystem
+from test_tools.fio.fio import Fio
+from test_tools.fio.fio_param import ReadWrite, IoEngine, VerifyMethod
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from test_utils.os_utils import sync
+from test_utils.size import Unit, Size
+
+
+start_size = Size(512, Unit.Byte).get_value()
+stop_size = Size(128, Unit.KibiByte).get_value()
+file_min_size = Size(4, Unit.KibiByte).get_value()
+file_max_size = Size(2, Unit.GibiByte).get_value()
+runtime = datetime.timedelta(days=5)
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core1", DiskTypeLowerThan("cache"))
+@pytest.mark.require_disk("core2", DiskTypeLowerThan("cache"))
+@pytest.mark.require_disk("core3", DiskTypeLowerThan("cache"))
+@pytest.mark.require_disk("core4", DiskTypeLowerThan("cache"))
+def test_data_integrity_5d_dss():
+    """
+        title: |
+          Data integrity test on three cas instances with different
+          file systems with duration time equal to 5 days
+        description: |
+          Create 3 cache instances on caches equal to 50GB
+          and cores equal to 150GB with different file systems
+        pass_criteria:
+            - System does not crash.
+            - All operations complete successfully.
+            - Data consistency is being preserved.
+    """
+    with TestRun.step("Prepare cas instances with filesystems"):
+        cas_instances = prepare_instances_5d_dss()
+
+    with TestRun.step("Run test workloads on filesystems with verification"):
+        run_workload_dss(cas_instances)
+
```

Comment:
I think that it would be nice to add flush CAS devices here and check if exported object's and core disk's  md5 sums are identical
In the end you can add stop CAS devices method

Reply:
Done.